learn/get-started/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Get Started
+---
+
+The best way to start your journey with Stellar is to create a [test network](./test-net.html) account and send your first transaction.
+
+In this tutorial we will be using our [JavaScript SDK](https://www.stellar.org/developers/js-stellar-sdk/learn/index.html) but you can use other [SDKs](https://www.stellar.org/developers/horizon/learn/#libraries) as well.
+
+## Installing `stellar-sdk`
+
+First create your NodeJS project if `package.json` file is not present in your working directory:
+```
+npm init
+```
+
+Then install our JavaScript SDK:
+```
+npm install --save stellar-sdk
+```
+
+## Creating a test network account
```

Comment:
Perhaps it might be better to split it into more small parts and number the steps. Also, instead of having one big code block, it might be better to split it up into multiple code blocks and at the end, show how the code all comes together.